exercises/secret-handshake/example.js

```diff
@@ -1,30 +1,24 @@
-export default function (handshake) {
-  const HANDSHAKE_COMMANDS = ['wink', 'double blink', 'close your eyes', 'jump', 'REVERSE'];
+const HANDSHAKE_COMMANDS = ['wink', 'double blink', 'close your eyes', 'jump', 'REVERSE'];
 
+export const secretHandshake = (handshake) => {
   if (typeof handshake !== 'number') {
     throw new Error('Handshake must be a number');
   }
 
-  this.commands = () => this.shakeWith;
+  const shakeWith = [];
 
-  this.calculateHandshake = (handshake) => {
-    const shakeWith = [];
+  for (let i = 0; i < HANDSHAKE_COMMANDS.length; i++) {
```

Comment:
I didn't write this algorithm, I just converted it from a method to a function and re-indented it.  I've made a more significant alteration to the algorithm by filtering over the array, instead of implementing a for loop or forEach loop.  Is this more acceptable?